docs/public/_redirects

```diff
@@ -8,6 +8,7 @@
 /r/x-license-eula https://mui.com/store/legal/mui-x-eula/ 302
 /r/x-license-key-installation https://mui.com/x/introduction/licensing/#license-key-installation 302
 /r/x-data-grid-no-dimensions https://mui.com/x/react-data-grid/layout/ 302
+/r/x-professional-support https://mui.com/x/introduction/support/#professional-support 302
```

Comment:
```suggestion
/r/x-technical-support https://mui.com/x/introduction/support/#technical-support 302
```

Reply:
Hey @mbrookes, I believe the idea of using "Professional" support is to contrast with the "Community" support.
We've been using "Professional" support in a lot of places by now.

Reply:
Ah, now I get the point that Matt wants to make.

+1 for the proposed change:

```suggestion
/r/x-technical-support https://mui.com/x/introduction/support/#technical-support 302
```

I have started to explore this problem a bit more in https://www.notion.so/mui-org/Professional-services-b9548d4fee2a4181a5787d373854603f, it aligns matter with the terminology I could find in the wild.

Reply:
I guessed that it was a mix up (mash up?) between professional services and technical support. What's (briefly) described at the linked page is technical support, not professional services (consultancy).

Reply:
> We've been using "Professional" support in a lot of places by now.

Then you might have a lot of places to fix it. 😁